Inc/C++Utilities/CppImports.hpp

```diff
@@ -16,7 +16,11 @@
 #include <array>
 #include <bitset>
 #include <ctype.h>
-#include <malloc.h>
+#ifdef __APPLE__
+#include <stdlib.h>  // macOS
+#else
+#include <malloc.h>  // Linux/Unix
+#endif
```

Comment:
This file is not taken into account in the mock, also protect this by using SIM_ON
```suggestion
#ifdef SIM_ON
#ifdef __APPLE__
#include <stdlib.h>  // macOS
#else
#include <malloc.h>  // Linux/Unix
#endif
#else
#include <malloc.h>
#endif
```
Just in case for some reason when compiling from a mac to the MCU the flag __APPLE__ is set for some unknown reason